packages/web5/test/dids/did_dht_test.dart

```diff
@@ -1,22 +1,23 @@
+import 'package:web5/src/dids/did.dart';
 import 'package:web5/src/dids/did_dht/did_dht.dart';
 import 'package:test/test.dart';
 
 void main() {
   group('DidDht', () {
-    test('should resolve with error if dookie DID', () async {
-      final resolutionResult = await DidDht.resolve('hi');
+    // test('should resolve with error if dookie DID', () async {
+    //   final resolutionResult = await DidDht.resolve('hi');
 
-      expect(resolutionResult.didDocument, isNull);
-      expect(
-        resolutionResult.didResolutionMetadata.error,
-        equals('invalidDid'),
-      );
-    });
+    //   expect(resolutionResult.didDocument, isNull);
+    //   expect(
+    //     resolutionResult.didResolutionMetadata.error,
+    //     equals('invalidDid'),
+    //   );
+    // });
```

Comment:
Do you want this here?

Reply:
ah thanks for reminding. removed in favor of global resolver test here: https://github.com/TBD54566975/web5-dart/commit/ee1c5ef1dbe564092be9ff6f138ef5ae059cab09

these tests are no longer applicable at for method specific resolvers because the method signature changed to expecting `Did` vs just a string. the global resolver takes a string